resources/js/features/achievements/components/CreateAchievementTicketMainRoot/CreateAchievementTicketForm/useCreateAchievementTicketForm.ts

```diff
@@ -0,0 +1,96 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+import { useLaravelReactI18n } from 'laravel-react-i18n';
+import { useForm } from 'react-hook-form';
+import { z } from 'zod';
+
+import { toastMessage } from '@/common/components/+vendor/BaseToaster';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { TicketType } from '@/common/utils/generatedAppConstants';
+
+const createAchievementTicketFormSchema = z.object({
+  /** @see TicketType.php */
+  issue: z.enum(['DidNotTrigger', 'TriggeredAtWrongTime', 'NetworkIssue']),
+  emulator: z.string().min(1),
+  emulatorVersion: z.string().optional(),
+  core: z.string().optional(),
+  mode: z.enum(['hardcore', 'softcore']),
+  hash: z.string().min(1),
+  description: z
+    .string()
+    .min(25, { message: 'Please be more detailed in your description.' })
```

Comment:
This is almost pointless with the template. You can add a space anywhere in the text and hit submit without providing any information.

Reply:
I went ahead and decided to remove the template in latest.

I think if we're going to do a template, it'd probably make sense to build it into the form itself as actual templated fields, rather than a placeholder value the user has to go out of their way to replace.